acceptancetests/assess_persistent_storage.py

```diff
@@ -27,6 +33,116 @@
 __metaclass__ = type
 log = logging.getLogger("assess_persistent_storage")
 
+def assess_charm_deploy_and_remove_single_storage(client):
+    """This function tests charm deployment and removal with single
+       persistent storage unit.
+
+       Steps taken to the test:
+       - Deploy dummy-storage charm with a single block storage unit (ebs)
+         and a single filesystem storage unit (rootfs).
+       - Check charm status once the deployment is done.
+       - Check charm storage list once the deployment is done.
+       - Remove the deployed charm, the output should indicate that the
+         rootfs storage unit will be removed, ebs storage unit will be
+         detached.
+       - Check storage list again to ensure ebs storage units really exists.
+
+       :param client: ModelClient object to deploy the charm on.
+    """
+
+    charm_name = 'dummy-storage'
+    charm_path = local_charm_path(
+        charm=charm_name, juju_ver=client.version)
+    log.info(
+        '{} is going to be deployed with 1 ebs block storage unit and '\
+        '1 rootfs filesystem storage unit.'.format(charm_name))
+    log.info(
```

Comment:
Oh, I guess you mean line 59, I thought it's line 56. Yes, the command line logging itself is being removed. As mentioned, in get_juju_output() the args are in debug level which just need to be changed to info level.